.gitignore

```diff
@@ -24,3 +24,6 @@ dist
 *.sh
 .devcontainer/*
 .vscode/*
+
+# private enviroment overrides
+.env.lamp_private
```

Comment:
suggestion: other projects handle his by putting `.env` in the .gitignore file and having a `.env.template` file that gets committed. 

lamp never got around to implementing this, but it would probably be a good idea to match other tid applications.